src/ui/popup/settings-page/devtools.tsx

```diff
@@ -25,15 +38,25 @@ function getExistingDevToolsWindow() {
 
 async function openDevTools() {
     const devToolsWindow = await getExistingDevToolsWindow();
-    if (devToolsWindow) {
-        chrome.windows.update(devToolsWindow.id, {'focused': true});
+    if (!isMobile()) {
```

Comment:
It's better to not invert the expression in `if ... else` condition. So, make it `if (isMobile())`